actions/templates/list/index.js

```diff
@@ -279,7 +276,7 @@ function filter(templates, filterValues, field, filterType, subfield) {
       case 'array': {
         let templateFieldValues = template[field];
         if (subfield) {
-          templateFieldValues = Array.isArray(template[field]) ? template[field].map(item => item[subfield]) : template[field][subfield];
```

Comment:
Removed because this code is not reachable, none of the filter fields that have a type of `array` can be returned as anything other than arrays